pkgs/applications/misc/rofimoji/default.nix

```
@@ -25,6 +25,8 @@ buildPythonApplication rec {
     sha256 = "08ayndpifr04njpijc5n5ii5nvibfpab39p6ngyyj0pb43792a8j";
   };
 
+  patches = [ ./fix-pyxdg-version.patch ];
```

```suggestion
  postPatch = "substituteInPlace setup.py --replace 'pyxdg==0.26' 'pyxdg'";
```
I'd suggest this instead of a file patch, especially for this simple of a change. File patches are mildly fragile (they depend on context), and they add another file to the already-large ``nixpkgs`` repo.

Additionally, I suggest unpinning ``pyxdg`` entirely unless there's an explicit reason not to. This will prevent issues in the future if the version changes (though ``>=  > ==``), and lets the integration tests do their job of catching build issues in the python package set.

---

I hadn't thought about this. Good point.